x10/utils/http.py

```diff
@@ -115,11 +123,14 @@ async def send_get_request(
     model_class: Type[ApiResponseType],
     *,
     api_key: Optional[str] = None,
+    request_headers: Optional[Dict[str, str]] = None,
+    response_code_to_exception: Optional[Dict[int, Type[Exception]]] = None,
```

Comment:
I think we will convert this to a constant mapping to be applied to every response.